test_pegasus/aseg_concatenation.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
We should try to separate a bit better (perhaps by folder):
- machine dependent configurations
- patient configurations
- python fixed code

Another ides is to split based on time when the file/data is needed:
- configuration time
- flow execution time